src/Libraries/PythonNodeModelsWpf/PythonNodeModelsWpf.csproj

```diff
@@ -42,29 +42,21 @@
   <ItemGroup>
 	<PackageReference Include="AvalonEdit" Version="6.3.0.90" />
 	<PackageReference Include="FontAwesome.WPF" Version="4.7.0.9" />
-    <Reference Include="Microsoft.Practices.Prism">
-      <HintPath>..\..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)'=='net48' ">
     <Reference Include="PresentationCore">
-      <Private>False</Private>
+        <Private>False</Private>
     </Reference>
     <Reference Include="PresentationFramework">
-      <Private>False</Private>
+        <Private>False</Private>
     </Reference>
     <Reference Include="System.Xaml" />
     <Reference Include="WindowsBase" />
```

Comment:
Isn't this required for net48?

Reply:
Yes, it probably is required by net48
I just changed the spacing